pkgs/applications/editors/emacs/elisp-packages/manual-packages/lsp-bridge/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, trivialBuild
+, fetchFromGitHub
+, python310Packages
```

Comment:
Please use `python3` without minor qualifiers if that works.

Reply:
Thanks for the hint, done that.